README.md

```diff
@@ -28,14 +28,17 @@ const checkoutInstance = startCheckout(manifest, [options])
 
 **`manifest`** - server to server response from checkout session creation
 
-| Option       | Type       | Required | Default            | Description                                                                |
-| ------------ | ---------- | -------- | ------------------ | -------------------------------------------------------------------------- |
-| `id`         | `string`   | no       | `easypay-checkout` | The id of the HTML element where the Checkout form should be included.     |
-| `onSuccess`  | `function` | no       | `() => {}`         | Callback function to be called when the Checkout is finished successfully. |
-| `onError`    | `function` | no       | `() => {}`         | Callback function to be called on errors.                                  |
-| `onClose`    | `function` | no       | `undefined`        | Callback function to be called when the Checkout popup is closed.          |
-| `testing`    | `boolean`  | no       | `false`            | Whether to use the testing API (`true`) or the production one (`false`).   |
-| `display`(1) | `string`   | no       | `inline`           | The display style of the element that hosts the Checkout                   |
+| Option        | Type       | Required | Default            | Description                                                                |
+| ------------- | ---------- | -------- | ------------------ | -------------------------------------------------------------------------- |
+| `id`          | `string`   | no       | `easypay-checkout` | The id of the HTML element where the Checkout form should be included.     |
+| `onSuccess`   | `function` | no       | `() => {}`         | Callback function to be called when the Checkout is finished successfully. |
+| `onError`     | `function` | no       | `() => {}`         | Callback function to be called on errors.                                  |
+| `onClose`     | `function` | no       | `undefined`        | Callback function to be called when the Checkout popup is closed.          |
+| `testing`     | `boolean`  | no       | `false`            | Whether to use the testing API (`true`) or the production one (`false`).   |
+| `display`(1)  | `string`   | no       | `inline`           | The display style of the element that hosts the Checkout.                  |
+| `hideDetails` | `boolean`  | no       | `false`            | Whether to use the customer details hidden or not.                         |
+| `language`    | `string`   | no       | `undefined`        | The language to display the Checkout.                                      |
```

Comment:
Missing the proposition, like `The language in which to display the Checkout.`.
We should add the available options: `en` and `pt_PT`.

Reply:
en is the default?